RockLib.Configuration/Config.cs

```diff
@@ -108,55 +111,50 @@ public static void ResetRoot(IEnumerable<KeyValuePair<string, string>> additiona
         /// The base path for the <see cref="ConfigurationBuilder"/>, or null to use the value
         /// returned by the <see cref="Directory.GetCurrentDirectory"/> method.
         /// </param>
-        public static void SetBasePath(string basePath = null)
+        public static void SetBasePath(string? basePath = null)
         {
             _basePath = basePath ?? Directory.GetCurrentDirectory();
         }
 
-        private static IConfiguration GetDefaultRoot(IEnumerable<KeyValuePair<string, string>> additionalValues)
+        private static IConfiguration GetDefaultRoot(IEnumerable<KeyValuePair<string, string>>? additionalValues)
         {
             var builder = new ConfigurationBuilder();
 
             if (_basePath != null)
```

Comment:
`!= null` should be `is not null`

Reply:
For crying out...I thought I had searched for all of them! :)